RELEASES.md

```diff
@@ -0,0 +1,77 @@
+# Release Process
+This document explains the process to release new version and to address patch release.
+
+## Major release
+This refers to the release of new features.
+
+### Confirm the changelog and Create Release Note
+- Run the release script
+
+  This example assumes that `vX.Y.Z` will be released:
+  ```shell
+  make gen/release version=vX.Y.Z
+  ````
+
+  `RELEASE` file will be updated and `docs/content/en/blog/releases/vX.Y.Z.md` file will be created.
+
+- Push the above changes and Create a pull request to confirm the changelog.
+  You can confirm the changelog through the reviewing comment in pull request by GitHub Actions.
+  For more information, Please see [actions-gh-release](https://github.com/pipe-cd/actions-gh-release).
+
+- Update the content in `docs/content/en/blog/releases/vX.Y.Z.md` file based on changelog results.
+  Please refer to [this example](https://github.com/pipe-cd/pipecd/pull/3792/commits/2b59f7f2a492405bf6df905b3823b417e4a10c3e).
+
+  It is recommended to commit the above changes once here.
+
+### Generate document for new version
+- Run the release document script
+
+  This example assumes that `vX.Y.Z` will be released:
+  ```shell
+  make gen/release-docs version=vX.Y.Z
+  ````
+
+- Push the above changes and get a review and merge.
+
+### Cut a new release
+- Before cutting a new release, wait for all jobs in GitHub Actions to pass on master branch.
+
+- Create a tagged release. The release should start with "v" and be followed by the version number.
+
+- Create a branch from the tagged release, e.g. "release-X.Y". This will be used for the patch release.
+
+## Patch release
+This refers to the release of critical bug fixes.
```

Comment:
```suggestion
This refers to the release of critical bug fixes. \
```
nits

Reply:
Sorry, I'm not sure about this meaning. Could you explain that difference?

Reply:
Ah, I see. Thanks!